Cargo.toml

```
@@ -2,9 +2,11 @@
 name = "policy-server"
 version = "1.1.2"
 authors = [
+  "Kubewarden",
```

```suggestion
  "Kubewarden Developers <kubewarden@suse.de>"
```

Would match what we have in `kwctl`.Maybe something like `Kubewarden Developers (github @kubewarden/kubewarden-developers)` would fit more, but it seems that[ spdx expects only emails there](https://spdx.github.io/spdx-spec/v2.3/package-information/#763-examples)